lingua_franca/res/text/en-us/normalize.json

```diff
@@ -99,7 +99,78 @@
     "you'd've": "you would have",
     "you'll": "you will",
     "you're": "you are",
-    "you've": "you have"
+    "you've": "you have",
+    "I'm'a": "I am about to",
+    "I'm'o": "I am going to",
+    "I'll've": "I will have",
+    "I'd've": "I would have",
+    "Whatcha": "What are you",
+    "amn't": "am not",
```

Comment:
I agree, though recently had a debate about this contraction specifically in a friend group because it is apparently used more widely than I'd have thought (the consensus was about what you just said, but same is true of how "literally" sometimes means "not literally").